models/issues/issue.go

```diff
@@ -201,12 +201,12 @@ func (issue *Issue) IsTimetrackerEnabled(ctx context.Context) bool {
 }
 
 // GetPullRequest returns the issue pull request
-func (issue *Issue) GetPullRequest() (pr *PullRequest, err error) {
+func (issue *Issue) GetPullRequest(ctx context.Context) (pr *PullRequest, err error) {
```

Comment:
`templates/shared/issueicon.tmpl` need to be updated

Reply:
God catch! We really need better frontend tests.